.circleci/runbuild.js

```diff
@@ -6,17 +6,42 @@ const http = require('https');
 const inquirer = require('inquirer');
 const simpleGit = require('simple-git')();
 
+
+/*
+posts a request for a circleci workflow
+you can alternatively use a curl command like:
+
+curl -u ${CIRCLECI_TOKEN}: -X POST \
+  --header 'Content-Type: application/json' \
+  -d '{
+    "branch": "JSDK-2537_migrate_integration_tests",
```

Comment:
once this PR is merged should we be using "Master" for v2?

Reply:
Yes. You would specify branch: "master" - I will send out instruction on how to start the job with curl command.